Apps/Examples/Examples/All Examples/RestrictCoordinateBoundsExample.swift

```diff
@@ -22,7 +21,7 @@ public class RestrictCoordinateBoundsExample: UIViewController, ExampleProtocol
 
         // Center the camera on the bounds
         let camera = mapView.mapboxMap.camera(for: bounds, padding: .zero, bearing: 0, pitch: 0)
-        
+
         // Set the camera's center coordinate.
```

Comment:
Nits, looks like there were no real changes here so can they be removed so there is not change to this file